src/common-principles.md

```diff
@@ -97,6 +97,12 @@ and/or files (like `events.tsv`) are fully omitted *when they are unavailable or
 instead of specified with an `n/a` value, or included as an empty file
 (for example an empty `events.tsv` file with only the headers included).
 
+## Dataset naming
+
+BIDS does not prescribe a particular naming scheme for directories containing individual BIDS datasets.
+However, it is RECOMMENDED to use a short descriptive name that reflects the content of the dataset, avoid spaces in the name, and use hyphens to separate words.
```

Comment:
RECOMMENDED implies a warning. I would remove this line, unless you want to add a validator rule.

```suggestion
```

Reply:
It is our goal. It is more true in the schema validator than in the legacy validator.

Reply:
So -- I would not mind having  validator support for that.
Also I could use the "trick" you suggested below instead for now
```suggestion
However, we suggest to use a short descriptive name that reflects the content of the dataset, avoid spaces in the name, and use hyphens to separate words.
```

;-)  but I do not think we actually want to introduce such a new non-standardized notion as "we suggest" (here or below), do we ?

Reply:
Right, I don't really like these unenforceable recommendations. It makes it unclear what's spec and what's not.

Reply:
to me it is more of using an established in common principles terminology (REQUIRED, RECOMMENDED, OPTIONAL). In there `REQUIRED` and `OPTIONAL` are really two opposite ends pertinent to what spec mandates or allows for. RECOMMENDED also semantically means to me advisory similar to "we suggest" and does not necessarily imply that it is part of the specification "mandate".  But I am ok also to make it e.g. lowercase to somewhat emphasize that it is really not the RECOMMENDED as part of the specification. WDYT?

  ```suggestion
However, it is recommended to use a short descriptive name that reflects the content of the dataset, avoid spaces in the name, and use hyphens to separate words.
```

Reply:
That would be fine with me. Do we care about "hyphens" being overly restrictive, or are we okay leaving it to people's judgment? I often use underscores as well when naming directories.